package.json

```diff
@@ -59,10 +59,10 @@
     "demo": "http-server"
   },
   "dependencies": {
-    "vl-ui-core": "^4.0.0"
+    "vl-ui-core": "milieuinfo/webcomponent-vl-ui-core#e2e"
   },
   "peerDependencies": {
-    "vl-ui-core": "^4.0.0"
+    "vl-ui-core": "milieuinfo/webcomponent-vl-ui-core#e2e"
   },
```

Comment:
Ga je wachten met mergen en pas mergen als we in de e2e branches van alle componenten een release van vl-ui-core gebruiken? 
Als we releases maken met branches in de dependencies gaan we uiteindelijk releases overhouden die niet meer werken aangezien deze branches verwijderd gaan worden na een merge naar master.

Reply:
Inderdaad. Er is een ticket op op het einde al de vl-ui-core branches te vervangen en overal een release van te maken. Geen tussentijdse releases dus.

Reply:
Ok cava :)